checkov/kubernetes/checks/DropCapabilities.py

```diff
@@ -23,7 +23,7 @@ def scan_spec_conf(self, conf):
             if conf["securityContext"].get("capabilities"):
                 if conf["securityContext"]["capabilities"].get("drop"):
                     for d in conf["securityContext"]["capabilities"]["drop"]:
-                        if "ALL" in d or "NET_RAW" in d:
+                        if "ALL" in d or "all" in d or "NET_RAW" in d:
```

Comment:
perfect use case for `any()`
```suggestion
                        if any(cap in d for cap in ("ALL", "all", "NET_RAW")):
```